api/src/client/dynamics/housing/DynamicsHousingRegistrationClient.ts

```diff
@@ -1,31 +1,45 @@
 import {
   getPropertyInterestTypeIntegerFromTitle,
-  HotelMotelRegistrationClient,
+  HousingRegistrationClient,
   HousingRegistrationRequest,
   HousingRegistrationRequestResponse,
   HousingRegistrationStatus,
 } from "@client/dynamics/housing/types";
 import { LogWriterType } from "@libs/logWriter";
 import { parseDate } from "@shared/dateHelpers";
+import { PropertyInterestType } from "@shared/housing";
 import axios, { AxiosError } from "axios";
 
-export const DynamicsHotelMotelRegistrationClient = (
+export const DynamicsHousingRegistrationClient = (
   logWriter: LogWriterType,
   orgUrl: string
-): HotelMotelRegistrationClient => {
-  const getHotelMotelRegistration = async (
+): HousingRegistrationClient => {
+  const getHousingRegistration = async (
     accessToken: string,
     propertyInterestId: string,
-    buildingCount: number
+    buildingCount: number,
+    propertyInterestType: PropertyInterestType
```

Comment:
[dust] Is there a point we should transition from multiple positional arguments to passing in a single config object?

Reply:
This would be a good thing to add to the style guide, I think I'm fine with 4 but if it gets to five I'll pull it into an object